examples/chip-tool/commands/common/EchoCommand.cpp

```diff
@@ -78,20 +80,35 @@ void EchoCommand::ReceiveEcho(PacketBuffer * buffer) const
 CHIP_ERROR EchoCommand::Run(ChipDeviceController * dc, NodeId remoteId)
 {
     CHIP_ERROR err = CHIP_NO_ERROR;
+    ExchangeManager * em = nullptr;
+    ExchangeContext * ec = nullptr;
 
     err = NetworkCommand::Run(dc, remoteId);
     SuccessOrExit(err);
 
+    em = dc->GetExchangeManager();
+    VerifyOrExit(em != nullptr, err = CHIP_ERROR_NO_MEMORY);
+
+    ec = em->NewContext(remoteId, this);
+    VerifyOrExit(ec != nullptr, err = CHIP_ERROR_NO_MEMORY);
```

Comment:
Client (AKA controller) needs to manually create an `ExchangeContext` to send/receive messages.